inflammation-analysis.py

```diff
@@ -20,14 +20,14 @@ def main(args):
 
     for filename in infiles:
         inflammation_data = models.load_csv(filename)
-
```

Comment:
i see a "-" sign here but I don't know what it means...

Reply:
not in code, but python signal that something has been deleted